heat/core/manipulation.py

```diff
@@ -0,0 +1,64 @@
+from . import dndarray
+from . import stride_tricks
+
+__all__ = [
+    'expand_dims'
+]
+
+
+def expand_dims(a, axis):
+    """
+    Expand the shape of an array.
+
+    Insert a new axis that will appear at the axis position in the expanded array shape.
+
+    Parameters
+    ----------
+    a : ht.DNDarray
+        Input array to be expanded.
+    axis : int
+        Position in the expanded axes where the new axis is placed.
+
+    Returns
+    -------
+    res : ht.DNDarray
+        Output array. The number of dimensions is one greater than that of the input array.
+
+    Raises
+    ------
+    ValueError
+        If the axis is not in range of the axes.
+
+    Examples
+    --------
+    >>> x = ht.array([1,2])
+    >>> x.shape
+    (2,)
+
+    >>> y = np.expand_dims(x, axis=0)
```

Comment:
This should be changed to `ht.expand_dims(x, axis=0)`. Same goes for the other examples.

Reply:
Thanks for the remark